src/components/stepper/themes/stepper/stepper.base.scss

```diff
@@ -0,0 +1,137 @@
+@use '../../../../styles/common/component';
+@use '../../../../styles/utilities' as *;
+
+// STEPPER
+:host {
+    font-family: var(--igc-font-family);
+
+    --margin: #{rem(16px)};
+    --body-padding: #{rem(16px)};
+    --header-gap: #{rem(8px)};
+    --header-padding: #{rem(8px)};
+    --vertical-body-padding: var(--body-padding);
+
+    // Indicator
+    --indicator-size: #{rem(24px)};
+    --indicator-roundness: calc(var(--indicator-size) / 2);
+    --indicator-box-shadow-size: #{rem(1px)};
+
+    // Separator
+    --separator-size: #{rem(1px)};
+    --separator-type: dashed;
+    --separator-type--completed: solid;
+    --separator-position: calc(var(--header-padding) + (var(--indicator-size) / 2) - (var(--separator-size) / 2));
+    --separator-min-width: #{rem(36px)};
+    --separator-min-width--header: #{rem(4px)};
+    --separator-min-width--full: calc(var(--separator-min-width) + var(--separator-min-width--header));
+    --text-min-width: var(--indicator-size);
+
+    // Step min width
+    --header-min-width: calc(var(--indicator-size) + (var(--header-padding) * 2));
+    --step-min-width: calc(var(--header-min-width) + var(--separator-min-width--full));
+    --step-min-width--first-step: var(--header-min-width);
+}
+
+// STEPPER [HORIZONTAL]
+:host([orientation='horizontal']) {
+    --horizontal-content-disply: block;
+    --vertical-separator-visibility: none;
+    --horizontal-body-padding: #{rem(8px)};
+
+    display: grid;
+    /* stylelint-disable */
+    grid-template-columns: minmax(var(--step-min-width--first-step), max-content) repeat(calc(#{var(--steps-count)} - 1), minmax(var(--step-min-width), auto));
```

Comment:
```suggestion
    grid-template-columns: minmax(var(--step-min-width--first-step), max-content) repeat(calc(var(--steps-count) - 1), minmax(var(--step-min-width), auto));
```

Reply:
For this one, my editor(webstorm) is complaining it gives me an error message "Mismatched parameters ([auto-fill | <integer>] , <line-names>+)" However, I removed it anyway and everything is working correctly, I will ignore that message from webstorm, they tend to have some issues regarding error messages for CSS/SCSS.